install/ci-vm/ci-linux/ci/variables-sample

```diff
@@ -7,27 +7,19 @@
 maintenanceURL="http://foo.bar/maintenance/linux"
```

Comment:
Updated the comment, and since we no longer need variables_sample (variables already filled in as per startup-script, as now we don't have much direct control on GCP instances when compared to KVMs), I have renamed it to variables itself.